app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import DisplayValue from '../../../../../components/FormComponents/DisplayValue'
+
+const InformationItem = ({ label, value, showIfMissing }) => {
```

Comment:
Not particularly happy with this name, any other suggestions would be welcome.

Reply:
`<ValueWithLabel />`?

Reply:
Nice work on getting rid of the repetition

Reply:
There was already a `ValueWithLabel` which was used for displaying "Assigned Staff Members" on quick look so I've merged the two components as they were doing the same thing..